tests/pgsql/pgsql-7000-ips/test.yaml

```diff
@@ -0,0 +1,533 @@
+requires:
+   min-version: 8
+args:
+- -k none
+- --simulate-ips
+
+checks:
+- filter:
+    count: 37
+    match:
+      event_type: alert
+- filter:
+    count: 1
+    match:
+      alert.signature: Pgsql table delete
+      alert.signature_id: 2
+      app_proto: pgsql
+      direction: to_server
+      event_type: alert
+      payload_printable: ......./...O....user.ju-Test.database.Test.application_name.psql.client_encoding.UTF8..p...6SCRAM-SHA-256.... n,,n=,r=ROtF8e2Fme8+eORLNHTwkZaKp...lc=biws,r=ROtF8e2Fme8+eORLNHTwkZaKtpbEaXYJOnd3qt6QNCsAv0wj,p=I4V0zdtQqrxum6B+QzprHHC0nBD+mVtBWpc+arfXa+c=Q....BEGIN;.
+      pcap_cnt: 19
+      pgsql.request.simple_query: DELETE FROM new_table WHERE NAME='Remus';
+      pgsql.tx_id: 6
+      stream: 1
+      tx_id: 5
+- filter:
+    count: 1
+    match:
+      alert.signature: Pgsql table delete
+      alert.signature_id: 2
+      app_proto: pgsql
+      direction: to_server
+      event_type: alert
+      payload_printable: ......./...O....user.ju-Test.database.Test.application_name.psql.client_encoding.UTF8..p...6SCRAM-SHA-256.... n,,n=,r=ROtF8e2Fme8+eORLNHTwkZaKp...lc=biws,r=ROtF8e2Fme8+eORLNHTwkZaKtpbEaXYJOnd3qt6QNCsAv0wj,p=I4V0zdtQqrxum6B+QzprHHC0nBD+mVtBWpc+arfXa+c=Q....BEGIN;.Q....DELETE FROM new_table WHERE NAME='Remus';.
+      pcap_cnt: 22
+      pgsql.request.simple_query: DELETE FROM new_table WHERE NAME='Londubat';
```

Comment:
Why do we have `Londubat` in  pgsql.request.simple_query ? and `Remus` in `payload_printable` ?

Reply:
This is part of the possible bug that I mentioned in the other comment: in IPS mode, possibly because the engine keeps a bigger stream window (I suppose?), we get alerts triggering in weird situations.